program/src/entrypoint.rs

```diff
@@ -9,9 +9,9 @@ use {
 
 solana_program_entrypoint::entrypoint!(process_instruction);
 
-fn process_instruction(
+fn process_instruction<'a>(
     program_id: &Pubkey,
-    accounts: &[AccountInfo],
+    accounts: &'a [AccountInfo<'a>],
```

Comment:
Lifetimes can't be elided anymore given the `wrapped_mint_auth_signer_seeds` now used in the processor:

```rust
   M::post_initialize_extensions(
            wrapped_mint_account,
            wrapped_token_program_account,
            wrapped_mint_authority_account,
            &wrapped_mint_auth_signer_seeds,
        )?;
```

Reply:
That part wasn't the issue, it was the account infos in invoke_signed, which need to share the same lifetime according to the `invoke_signed` signature: https://github.com/anza-xyz/solana-sdk/blob/1dd727d4754b22157ce2b890b0f6dfe5237bc0d1/cpi/src/lib.rs#L253

I simplified this a bit if you want to take a look: https://github.com/joncinque/token-wrap/tree/add-metadata-ext-jon